interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -142,8 +159,26 @@ namespace cling {
     return NPath;
   }
 
-  std::string
-  DynamicLibraryManager::lookupLibrary(llvm::StringRef libStem) const {
+  ///\returns true if substitution was successful.
+  static bool subst(llvm::StringRef value, std::string find,
+                    llvm::StringRef substWith, std::string& result) {
```

Comment:
The return value of this function isn't ever used. Consider
```c++
static std::string substFront(llvm::StringRef value, llvm::StringRef pattern, llvm::StringRef replacement);
```

Reply:
Yeah, good point. Fixed.